src/internal/config/config.go

```diff
@@ -84,11 +91,22 @@ type Service struct {
 	lock *sync.RWMutex
 }
 
-// BackoffCfg represents values used to configure
-// reconnecting hook when http errors occur.
+// backoffCfg represents values used to configure
```

Comment:
The comment requires some fixes I guess.

Reply:
You can add something about it as "a file storage representation of BackoffCfg" or something.